<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-3BVFOOV" failures="1" tests="7" name="testng.AnnotationsExample" time="0.043" errors="0" timestamp="2021-04-12T11:30:33 IST" skipped="0">
  <testcase classname="testng.AnnotationsExample" name="amethod4" time="0.011"/>
  <system-out/>
  <testcase classname="testng.AnnotationsExample" name="bmethod1" time="0.003"/>
  <system-out/>
  <testcase classname="testng.AnnotationsExample" name="signIn" time="0.003"/>
  <system-out/>
  <testcase classname="testng.AnnotationsExample" name="zmethod2" time="0.013">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at testng.AnnotationsExample.zmethod2(AnnotationsExample.java:84)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </failure>
  </testcase> <!-- zmethod2 -->
  <system-out/>
  <testcase classname="testng.AnnotationsExample" name="method3" time="0.005"/>
  <system-out/>
  <testcase classname="testng.AnnotationsExample" name="signIn" time="0.004"/>
  <system-out/>
  <testcase classname="testng.AnnotationsExample" name="signIn" time="0.004"/>
  <system-out/>
</testsuite> <!-- testng.AnnotationsExample -->
